src/wolfkrow/core/settings.yaml

```diff
@@ -14,4 +14,17 @@ deadline:
     ExtraInfo0: "$SHOW"
     ExtraInfo1: "$WORKSPACE"
     ExtraInfo2: "$WORKSPACE_PATH"
-    UserName: "$USER"
\ No newline at end of file
+    UserName: "$USER"
+
+  environment_inclusion_filters:
+    - TK_* # Toolkit environment variables
+    - *OCIO* # Include everything relating to ocio
+    - WOLFKROW* # Wolfkrow environment variables
+  environment_exclusion_filters:
+  environment_inclusion_list:
+    - PYTHONPATH
+    - PATH
+  environment_exclusion_list:
+
+nuke_submitter:
+  temp_dir: "{project_root}/wolfkrow_temp"
```

Comment:
This assumes there is a project_root replacement has been defined... WE should probably set this up so that the default implementation works out of the box. 
Change this to `temp_dir: "$TEMP/wolfkrow_temp` 

Note: This will typically fail though because `$TEMP` usually refers to a local drive, and wolfkrow executes on the farm. Maybe we should add a `WOLFKROW_TEMP` environment variables or entry in the settings file...